build-tools/packages/version-tools/README.md

```diff
@@ -125,7 +125,7 @@ EXAMPLES
   $ fluv autocomplete --refresh-cache
 ```
 
-_See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v2.3.9/src/commands/autocomplete/index.ts)_
+_See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v2.3.9/lib/commands/autocomplete/index.ts)_
```

Comment:
These changes seem wrong... can you make sure you run `pnpm i` again and rebuild to see if these go away?